pkg/plugins/hybrid/v1alpha/scaffolds/internal/templates/dockerfile.go

```diff
@@ -63,13 +63,14 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
 
 ENV HOME=/opt/helm \
     USER_NAME=helm \
-    USER_UID=1001
+    USER_UID=1001 \
+	G_ID=0
 
 RUN echo "${USER_NAME}:x:${USER_UID}:0:${USER_NAME} user:${HOME}:/sbin/nologin" >> /etc/passwd
 
 # Copy necessary files with the right permissions
-COPY --chown=${USER_UID}:${USER_UID} watches.yaml ${HOME}/watches.yaml
-COPY --chown=${USER_UID}:${USER_UID} helm-charts  ${HOME}/helm-charts
+COPY --chown=${USER_UID}:${G_ID} watches.yaml ${HOME}/watches.yaml
+COPY --chown=${USER_UID}:${G_ID} helm-charts  ${HOME}/helm-charts
```

Comment:
Nit: Maybe just hardcode `0` instead of providing a variable for this? We're already hardcoding `0` for the gid in the `/etc/passwd` line.

I think the hardcoding makes it less readily changable, which is a better indication that this isn't meant to be changed.